src/Native/gen-buildsys-clang.sh

```diff
@@ -3,13 +3,14 @@
 # This file invokes cmake and generates the build system for Clang.
 #
 
-if [ $# -lt 4 -o $# -gt 6 ]
+if [ $# -lt 4 -o $# -gt 7 ]
 then
   echo "Usage..."
-  echo "gen-buildsys-clang.sh <path to top level CMakeLists.txt> <ClangMajorVersion> <ClangMinorVersion> <Architecture> [build flavor] [cmakeargs]"
+  echo "gen-buildsys-clang.sh <path to top level CMakeLists.txt> <ClangMajorVersion> <ClangMinorVersion> <Architecture> [build flavor] [Static_Curl] [cmakeargs]"
```

Comment:
I don't think we should add this but instead just pass `-DCMAKE_STATIC_CURL` from build.sh or handle this the way we handle things like `LLDB_LIB_DIR`.  I feel like the way this patch is structured, it would do the wrong thing if a single additional cmake arg was passed.